src/test/java/org/eclipse/uprotocol/cloudevent/validate/CloudEventValidatorTest.java

```diff
@@ -60,7 +61,7 @@ void test_get_a_publish_cloud_event_validator() {
     @DisplayName("Test get a notification cloud event validator")
     void test_get_a_notification_cloud_event_validator() {
         CloudEventBuilder builder = buildBaseCloudEventBuilderForTest().withExtension("sink", "//bo.cloud/petapp")
-                .withType("pub.v1");
+                .withType("not.v1");
```

Comment:
not.v1???? IMHO not.v1 is so so confusing.

Reply:
There is a new message type called notification, this is what Manohar and Ying also wanted. So in lieu of inference if the message is of type publish vs notification, it is explicit.